### ISSUE TITLE
Custom DBT not working for SQL Server destination

### ISSUE THREAD

**Issue by meta-research-jack** [airbytehq/airbyte#21114]
## Environment
- **Airbyte version**:  0.40.26
- **OS Version / Instance**: Windows 10
- **Deployment**: Docker deploy env
- **Source Connector and version**: airbyte/source-file 0.2.31
- **Destination Connector and version**:  airbyte/destination-mssql 0.1.22
- **Step where error happened**: During custom transformation

## Current Behavior
I wanted to perform custom transformation after data replication to the destination database table. But the custom transformation did not work.

## Expected Behavior
Custom DBT should work after Airbyte clone my dbt project from Github (project tested can confirm it's working)

## Logs
`2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value HTTPS
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value GCS
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value S3
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value AzBlob, $.storage_account: is missing but it is required
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value SSH, $.password: object found, string expected
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value SCP, $.password: object found, string expected
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.password: object found, string expected
2023-01-06 15:36:50 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value local
2023-01-06 15:36:51 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/13/0/logs.log
2023-01-06 15:36:51 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.40.26
2023-01-06 15:36:51 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):172 - Attempt 0 to save workflow id for cancellation
2023-01-06 15:36:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:36:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- START CHECK -----
2023-01-06 15:36:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:36:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/source-file:0.2.31 exists...
2023-01-06 15:36:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/source-file:0.2.31 was found locally.
2023-01-06 15:36:52 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = source-file-check-13-0-pmxyt with resources io.airbyte.config.ResourceRequirements@1034c6e7[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:36:52 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0 --log-driver none --name source-file-check-13-0-pmxyt --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=true -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/source-file:0.2.31 -e AUTO_DETECT_SCHEMA=false -e AIRBYTE_VERSION=0.40.26 -e WORKER_JOB_ID=13 airbyte/source-file:0.2.31 check --config source_config.json
2023-01-06 15:36:52 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):97 - Reading messages from protocol version 0.2.0
2023-01-06 15:36:57 [33mWARN[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):117 - ignoring unsupported keyword arguments: ['connect_kwargs']
2023-01-06 15:37:00 [33mWARN[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):117 - ignoring unsupported keyword arguments: ['connect_kwargs']
2023-01-06 15:37:03 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Check succeeded
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- END CHECK -----
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:04 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-01-06 15:37:04 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/13/0/logs.log
2023-01-06 15:37:04 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.40.26
2023-01-06 15:37:04 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):172 - Attempt 0 to save workflow id for cancellation
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- START CHECK -----
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/destination-mssql:0.1.22 exists...
2023-01-06 15:37:04 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/destination-mssql:0.1.22 was found locally.
2023-01-06 15:37:04 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = destination-mssql-check-13-0-iidws with resources io.airbyte.config.ResourceRequirements@1034c6e7[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:37:04 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0 --log-driver none --name destination-mssql-check-13-0-iidws --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=true -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mssql:0.1.22 -e AUTO_DETECT_SCHEMA=false -e AIRBYTE_VERSION=0.40.26 -e WORKER_JOB_ID=13 airbyte/destination-mssql:0.1.22 check --config source_config.json
2023-01-06 15:37:04 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):97 - Reading messages from protocol version 0.2.0
2023-01-06 15:37:08 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - starting destination: class io.airbyte.integrations.destination.mssql.MSSQLDestination
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - integration args: {check=null, config=source_config.json}
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Command: CHECK
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Integration config: IntegrationConfig{command=CHECK, configPath='source_config.json', catalogPath='null', statePath='null'}
2023-01-06 15:37:09 [33mWARN[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):117 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2023-01-06 15:37:09 [33mWARN[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):117 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Starting connection with method: NO_TUNNEL
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - HikariPool-1 - Starting...
2023-01-06 15:37:09 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - HikariPool-1 - Start completed.
2023-01-06 15:37:12 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - HikariPool-1 - Shutdown initiated...
2023-01-06 15:37:12 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - HikariPool-1 - Shutdown completed.
2023-01-06 15:37:12 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination
2023-01-06 15:37:12 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - completed destination: class io.airbyte.integrations.destination.mssql.MSSQLDestination
2023-01-06 15:37:13 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-01-06 15:37:13 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:13 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- END CHECK -----
2023-01-06 15:37:13 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:14 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/13/0/logs.log
2023-01-06 15:37:14 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.40.26
2023-01-06 15:37:14 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):172 - Attempt 0 to save workflow id for cancellation
2023-01-06 15:37:15 [32mINFO[m i.a.c.f.EnvVariableFeatureFlags(getEnvOrDefault):64 - Using default value for environment variable LOG_CONNECTOR_MESSAGES: 'false'
2023-01-06 15:37:15 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1161 - Using default value for environment variable METRIC_CLIENT: ''
2023-01-06 15:37:15 [33mWARN[m i.a.m.l.MetricClientFactory(initialize):60 - Metric client is already initialized to 
2023-01-06 15:37:15 [32mINFO[m i.a.c.f.EnvVariableFeatureFlags(getEnvOrDefault):64 - Using default value for environment variable LOG_CONNECTOR_MESSAGES: 'false'
2023-01-06 15:37:15 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):142 - start sync worker. job id: 13 attempt id: 0
2023-01-06 15:37:15 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):157 - configured sync modes: {null.imported_data=full_refresh - overwrite}
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- START REPLICATION -----
2023-01-06 15:37:15 [32mINFO[m i.a.w.i.DefaultAirbyteDestination(start):72 - Running destination...
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/destination-mssql:0.1.22 exists...
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/destination-mssql:0.1.22 was found locally.
2023-01-06 15:37:15 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = destination-mssql-write-13-0-qnapz with resources io.airbyte.config.ResourceRequirements@71a9c604[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:37:15 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0 --log-driver none --name destination-mssql-write-13-0-qnapz --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=true -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mssql:0.1.22 -e AUTO_DETECT_SCHEMA=false -e AIRBYTE_VERSION=0.40.26 -e WORKER_JOB_ID=13 airbyte/destination-mssql:0.1.22 write --config destination_config.json --catalog destination_catalog.json
2023-01-06 15:37:15 [32mINFO[m i.a.w.i.VersionedAirbyteMessageBufferedWriterFactory(createWriter):33 - Writing messages to protocol version 0.2.0
2023-01-06 15:37:15 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):97 - Reading messages from protocol version 0.2.0
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/source-file:0.2.31 exists...
2023-01-06 15:37:15 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/source-file:0.2.31 was found locally.
2023-01-06 15:37:15 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = source-file-read-13-0-ojwrb with resources io.airbyte.config.ResourceRequirements@22d9bec4[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:37:15 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0 --log-driver none --name source-file-read-13-0-ojwrb --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=true -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/source-file:0.2.31 -e AUTO_DETECT_SCHEMA=false -e AIRBYTE_VERSION=0.40.26 -e WORKER_JOB_ID=13 airbyte/source-file:0.2.31 read --config source_config.json --catalog source_catalog.json
2023-01-06 15:37:15 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):97 - Reading messages from protocol version 0.2.0
2023-01-06 15:37:15 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromDstRunnable$4):260 - Destination output thread started.
2023-01-06 15:37:15 [32mINFO[m i.a.w.g.DefaultReplicationWorker(replicate):234 - Waiting for source and destination threads to complete.
2023-01-06 15:37:15 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$6):310 - Replication thread started.
2023-01-06 15:37:20 [43mdestination[0m > starting destination: class io.airbyte.integrations.destination.mssql.MSSQLDestination
2023-01-06 15:37:21 [43mdestination[0m > integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}
2023-01-06 15:37:21 [43mdestination[0m > Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination
2023-01-06 15:37:21 [43mdestination[0m > Command: WRITE
2023-01-06 15:37:21 [43mdestination[0m > Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}
2023-01-06 15:37:21 [43mdestination[0m > Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2023-01-06 15:37:21 [43mdestination[0m > Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2023-01-06 15:37:21 [43mdestination[0m > Starting connection with method: NO_TUNNEL
2023-01-06 15:37:22 [43mdestination[0m > HikariPool-1 - Starting...
2023-01-06 15:37:22 [43mdestination[0m > HikariPool-1 - Start completed.
2023-01-06 15:37:22 [43mdestination[0m > Write config: WriteConfig{streamName=imported_data, namespace=null, outputSchemaName=dbo, tmpTableName=_airbyte_tmp_zgr_imported_data, outputTableName=_airbyte_raw_imported_data, syncMode=overwrite}
2023-01-06 15:37:22 [43mdestination[0m > class io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer started.
2023-01-06 15:37:22 [43mdestination[0m > Preparing tmp tables in destination started for 1 streams
2023-01-06 15:37:22 [43mdestination[0m > Preparing tmp table in destination started for stream imported_data. schema: dbo, tmp table name: _airbyte_tmp_zgr_imported_data
2023-01-06 15:37:23 [44msource[0m > Reading imported_data (sftp://data.csv)...
2023-01-06 15:37:23 [44msource[0m > ignoring unsupported keyword arguments: ['connect_kwargs']
2023-01-06 15:37:24 [43mdestination[0m > Preparing tables in destination completed.
2023-01-06 15:37:28 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$6):350 - Source has no more messages, closing connection.
2023-01-06 15:37:28 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$6):359 - Total records read: 39 (4 KB)
2023-01-06 15:37:28 [32mINFO[m i.a.w.g.DefaultReplicationWorker(replicate):239 - One of source or destination thread complete. Waiting on the other.
2023-01-06 15:37:28 [43mdestination[0m > Airbyte message consumer: succeeded.
2023-01-06 15:37:28 [43mdestination[0m > executing on success close procedure.
2023-01-06 15:37:28 [43mdestination[0m > Flushing imported_data: 39 records (16 KB)
2023-01-06 15:37:29 [43mdestination[0m > No onDestinationCloseOperations required for this destination.
2023-01-06 15:37:29 [43mdestination[0m > Finalizing tables in destination started for 1 streams
2023-01-06 15:37:29 [43mdestination[0m > Finalizing stream imported_data. schema dbo, tmp table _airbyte_tmp_zgr_imported_data, final table _airbyte_raw_imported_data
2023-01-06 15:37:29 [43mdestination[0m > Executing finalization of tables.
2023-01-06 15:37:30 [43mdestination[0m > Finalizing tables in destination completed.
2023-01-06 15:37:30 [43mdestination[0m > Cleaning tmp tables in destination started for 1 streams
2023-01-06 15:37:30 [43mdestination[0m > Cleaning tmp table in destination started for stream imported_data. schema dbo, tmp table name: _airbyte_tmp_zgr_imported_data
2023-01-06 15:37:30 [43mdestination[0m > Cleaning tmp tables in destination completed.
2023-01-06 15:37:30 [43mdestination[0m > Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination
2023-01-06 15:37:30 [43mdestination[0m > completed destination: class io.airbyte.integrations.destination.mssql.MSSQLDestination
2023-01-06 15:37:31 [32mINFO[m i.a.w.g.DefaultReplicationWorker(replicate):241 - Source and destination threads complete.
2023-01-06 15:37:31 [32mINFO[m i.a.w.g.DefaultReplicationWorker(prepStateForLaterSaving):503 - Source did not output any state messages
2023-01-06 15:37:31 [33mWARN[m i.a.w.g.DefaultReplicationWorker(prepStateForLaterSaving):514 - State capture: No state retained.
2023-01-06 15:37:31 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):434 - sync summary: {
  "status" : "completed",
  "recordsSynced" : 39,
  "bytesSynced" : 4178,
  "startTime" : 1673019435347,
  "endTime" : 1673019451183,
  "totalStats" : {
    "recordsEmitted" : 39,
    "bytesEmitted" : 4178,
    "sourceStateMessagesEmitted" : 0,
    "destinationStateMessagesEmitted" : 0,
    "recordsCommitted" : 39,
    "meanSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "meanSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "replicationStartTime" : 1673019435347,
    "replicationEndTime" : 1673019451182,
    "sourceReadStartTime" : 1673019435480,
    "sourceReadEndTime" : 1673019448903,
    "destinationWriteStartTime" : 1673019435724,
    "destinationWriteEndTime" : 1673019451182
  },
  "streamStats" : [ {
    "streamName" : "imported_data",
    "stats" : {
      "recordsEmitted" : 39,
      "bytesEmitted" : 4178,
      "recordsCommitted" : 39
    }
  } ]
}
2023-01-06 15:37:31 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):435 - failures: [ ]
2023-01-06 15:37:31 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-01-06 15:37:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- END REPLICATION -----
2023-01-06 15:37:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:31 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):205 - sync summary: io.airbyte.config.StandardSyncOutput@13cdb50d[standardSyncSummary=io.airbyte.config.StandardSyncSummary@189c5729[status=completed,recordsSynced=39,bytesSynced=4178,startTime=1673019435347,endTime=1673019451183,totalStats=io.airbyte.config.SyncStats@7fc749a7[recordsEmitted=39,bytesEmitted=4178,sourceStateMessagesEmitted=0,destinationStateMessagesEmitted=0,recordsCommitted=39,meanSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBetweenStateMessageEmittedandCommitted=0,meanSecondsBetweenStateMessageEmittedandCommitted=0,replicationStartTime=1673019435347,replicationEndTime=1673019451182,sourceReadStartTime=1673019435480,sourceReadEndTime=1673019448903,destinationWriteStartTime=1673019435724,destinationWriteEndTime=1673019451182,additionalProperties={}],streamStats=[io.airbyte.config.StreamSyncStats@2581e055[streamName=imported_data,streamNamespace=<null>,stats=io.airbyte.config.SyncStats@5b9a4c35[recordsEmitted=39,bytesEmitted=4178,sourceStateMessagesEmitted=<null>,destinationStateMessagesEmitted=<null>,recordsCommitted=39,meanSecondsBeforeSourceStateMessageEmitted=<null>,maxSecondsBeforeSourceStateMessageEmitted=<null>,maxSecondsBetweenStateMessageEmittedandCommitted=<null>,meanSecondsBetweenStateMessageEmittedandCommitted=<null>,replicationStartTime=<null>,replicationEndTime=<null>,sourceReadStartTime=<null>,sourceReadEndTime=<null>,destinationWriteStartTime=<null>,destinationWriteEndTime=<null>,additionalProperties={}]]]],normalizationSummary=<null>,webhookOperationSummary=<null>,state=<null>,outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@21456d70[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@640a928e[stream=io.airbyte.protocol.models.AirbyteStream@4e234512[name=imported_data,jsonSchema={"type":"object","$schema":"http://json-schema.org/draft-07/schema#","properties":{"DateTime":{"type":["string","null"]},"SiteCode":{"type":["string","null"]},"TotalTransactionQty":{"type":["number","null"]},"TotalTransactionValue":{"type":["number","null"]}}},supportedSyncModes=[full_refresh],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[],namespace=<null>,additionalProperties={}],syncMode=full_refresh,cursorField=[],destinationSyncMode=overwrite,primaryKey=[],additionalProperties={}]],additionalProperties={}],failures=[]]
2023-01-06 15:37:31 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):210 - Sync summary length: 2352
2023-01-06 15:37:31 [32mINFO[m i.a.c.t.TemporalUtils(withBackgroundHeartbeat):283 - Stopping temporal heartbeating...
2023-01-06 15:37:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/13/0/logs.log
2023-01-06 15:37:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.40.26
2023-01-06 15:37:32 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):172 - Attempt 0 to save workflow id for cancellation
2023-01-06 15:37:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:32 [32mINFO[m i.a.w.n.DefaultNormalizationRunner(runProcess):121 - Running with normalization version: airbyte/normalization-mssql:0.2.25
2023-01-06 15:37:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- START DEFAULT NORMALIZATION -----
2023-01-06 15:37:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/normalization-mssql:0.2.25 exists...
2023-01-06 15:37:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/normalization-mssql:0.2.25 was found locally.
2023-01-06 15:37:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = normalization-mssql-normalize-13-0-hoayc with resources io.airbyte.config.ResourceRequirements@5807fb22[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:37:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0/normalize --log-driver none --name normalization-mssql-normalize-13-0-hoayc --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e AIRBYTE_VERSION=0.40.26 airbyte/normalization-mssql:0.2.25 run --integration-type mssql --config destination_config.json --catalog destination_catalog.json
2023-01-06 15:37:34 [42mnormalization[0m > Running: transform-config --config destination_config.json --integration-type mssql --out /data/13/0/normalize
2023-01-06 15:37:35 [42mnormalization[0m > Namespace(config='destination_config.json', integration_type=<DestinationType.MSSQL: 'mssql'>, out='/data/13/0/normalize')
2023-01-06 15:37:35 [42mnormalization[0m > transform_mssql
2023-01-06 15:37:35 [42mnormalization[0m > Running: transform-catalog --integration-type mssql --profile-config-dir /data/13/0/normalize --catalog destination_catalog.json --out /data/13/0/normalize/models/generated/ --json-column _airbyte_data
2023-01-06 15:37:36 [42mnormalization[0m > Processing destination_catalog.json...
2023-01-06 15:37:36 [42mnormalization[0m >   Generating airbyte_ctes/dbo/imported_data_ab1.sql from imported_data
2023-01-06 15:37:36 [42mnormalization[0m >   Generating airbyte_ctes/dbo/imported_data_ab2.sql from imported_data
2023-01-06 15:37:36 [42mnormalization[0m >   Generating airbyte_ctes/dbo/imported_data_ab3.sql from imported_data
2023-01-06 15:37:36 [42mnormalization[0m >   Adding drop table hook for imported_data_scd to imported_data
2023-01-06 15:37:36 [42mnormalization[0m >   Generating airbyte_tables/dbo/imported_data.sql from imported_data
2023-01-06 15:37:36 [42mnormalization[0m > detected no config file for ssh, assuming ssh is off.
2023-01-06 15:37:40 [42mnormalization[0m >            [--event-buffer-size EVENT_BUFFER_SIZE]
2023-01-06 15:37:40 [42mnormalization[0m >   --event-buffer-size EVENT_BUFFER_SIZE
2023-01-06 15:37:40 [32mINFO[m i.a.w.n.NormalizationAirbyteStreamFactory(filterOutAndHandleNonAirbyteMessageLines):104 - 
2023-01-06 15:37:40 [42mnormalization[0m > DBT >=1.0.0 detected; using 10K event buffer size
2023-01-06 15:37:40 [32mINFO[m i.a.w.n.NormalizationAirbyteStreamFactory(filterOutAndHandleNonAirbyteMessageLines):104 - 
2023-01-06 15:37:43 [42mnormalization[0m > Running with dbt=1.0.0
2023-01-06 15:37:43 [42mnormalization[0m > Partial parse save file not found. Starting full parse.
2023-01-06 15:37:45 [42mnormalization[0m > [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 2 unused configuration paths:
- models.airbyte_utils.generated.airbyte_views
- models.airbyte_utils.generated.airbyte_incremental

2023-01-06 15:37:45 [42mnormalization[0m > Found 4 models, 0 tests, 0 snapshots, 0 analyses, 601 macros, 0 operations, 0 seed files, 1 source, 0 exposures, 0 metrics
2023-01-06 15:37:49 [42mnormalization[0m > Concurrency: 8 threads (target='prod')
2023-01-06 15:37:49 [42mnormalization[0m > 1 of 1 START table model dbo.imported_data.............................................................................. [RUN]
2023-01-06 15:37:51 [42mnormalization[0m > 1 of 1 OK created table model dbo.imported_data......................................................................... [[32mOK[0m in 2.67s]
2023-01-06 15:37:51 [42mnormalization[0m > Finished running 1 table model in 6.90s.
2023-01-06 15:37:51 [42mnormalization[0m > [32mCompleted successfully[0m
2023-01-06 15:37:51 [42mnormalization[0m > Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
2023-01-06 15:37:52 [32mINFO[m i.a.w.n.DefaultNormalizationRunner(close):186 - Terminating normalization process...
2023-01-06 15:37:52 [32mINFO[m i.a.w.n.DefaultNormalizationRunner(close):199 - Normalization process successfully terminated.
2023-01-06 15:37:52 [32mINFO[m i.a.w.g.DefaultNormalizationWorker(run):94 - Normalization executed in 20 seconds for job 13.
2023-01-06 15:37:52 [32mINFO[m i.a.w.g.DefaultNormalizationWorker(run):106 - Normalization summary: io.airbyte.config.NormalizationSummary@5f1e8bf7[startTime=1673019452090,endTime=1673019472949,failures=[]]
2023-01-06 15:37:52 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-01-06 15:37:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- END DEFAULT NORMALIZATION -----
2023-01-06 15:37:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:52 [32mINFO[m i.a.c.t.TemporalUtils(withBackgroundHeartbeat):283 - Stopping temporal heartbeating...
2023-01-06 15:37:53 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/13/0/logs.log
2023-01-06 15:37:53 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.40.26
2023-01-06 15:37:53 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):172 - Attempt 0 to save workflow id for cancellation
2023-01-06 15:37:53 [32mINFO[m i.a.w.g.DbtTransformationWorker(run):58 - Running dbt transformation.
2023-01-06 15:37:53 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:53 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- START DBT TRANSFORMATION -----
2023-01-06 15:37:53 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:37:53 [32mINFO[m i.a.w.n.DefaultNormalizationRunner(runProcess):121 - Running with normalization version: airbyte/normalization-mssql:0.2.25
2023-01-06 15:37:53 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/normalization-mssql:0.2.25 exists...
2023-01-06 15:37:53 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/normalization-mssql:0.2.25 was found locally.
2023-01-06 15:37:53 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = normalization-mssql-normalize-13-0-vzuki with resources io.airbyte.config.ResourceRequirements@38868596[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:37:53 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0/transform --log-driver none --name normalization-mssql-normalize-13-0-vzuki --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e AIRBYTE_VERSION=0.40.26 airbyte/normalization-mssql:0.2.25 configure-dbt --integration-type mssql --config destination_config.json --git-repo https://github.com/meta-research-jack/dbt2.git
2023-01-06 15:37:54 [42mnormalization[0m > Running: git clone --depth 5 --single-branch  $GIT_REPO git_repo
2023-01-06 15:37:56 [42mnormalization[0m > Last 5 commits in git_repo:
2023-01-06 15:37:56 [42mnormalization[0m > 7518d5a use airbyte dbt
2023-01-06 15:37:56 [42mnormalization[0m > f3d5bb4 change to table
2023-01-06 15:37:56 [42mnormalization[0m > 23f9399 added files
2023-01-06 15:37:56 [42mnormalization[0m > /data/13/0/transform
2023-01-06 15:37:56 [42mnormalization[0m > Running: transform-config --config destination_config.json --integration-type mssql --out /data/13/0/transform
2023-01-06 15:37:57 [42mnormalization[0m > Namespace(config='destination_config.json', integration_type=<DestinationType.MSSQL: 'mssql'>, out='/data/13/0/transform')
2023-01-06 15:37:57 [42mnormalization[0m > transform_mssql
2023-01-06 15:37:57 [42mnormalization[0m > Cloning into 'git_repo'...
2023-01-06 15:37:57 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if fishtownanalytics/dbt:1.0.0 exists...
2023-01-06 15:37:57 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - fishtownanalytics/dbt:1.0.0 was found locally.
2023-01-06 15:37:57 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = dbt-custom-13-0-wctqq with resources io.airbyte.config.ResourceRequirements@38868596[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:37:57 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/0/transform --log-driver none --name dbt-custom-13-0-wctqq --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e AIRBYTE_VERSION=0.40.26 --entrypoint /bin/bash fishtownanalytics/dbt:1.0.0 entrypoint.sh run
2023-01-06 15:38:00 [45mdbt[0m > Running from /data/13/0/transform/git_repo
2023-01-06 15:38:00 [45mdbt[0m > detected no config file for ssh, assuming ssh is off.
2023-01-06 15:38:00 [45mdbt[0m > Running: dbt run --profiles-dir=/data/13/0/transform --project-dir=/data/13/0/transform/git_repo
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04  Error importing adapter: No module named 'dbt.adapters.sqlserver'
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04  Encountered an error while reading profiles:
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04    ERROR: Runtime Error
2023-01-06 15:38:04 [45mdbt[0m >   Credentials in profile "normalize", target "prod" invalid: Runtime Error
2023-01-06 15:38:04 [45mdbt[0m >     Could not find adapter type sqlserver!
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04  Defined profiles:
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04   - normalize
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04  
2023-01-06 15:38:04 [45mdbt[0m > For more information on configuring profiles, please consult the dbt docs:
2023-01-06 15:38:04 [45mdbt[0m > 
2023-01-06 15:38:04 [45mdbt[0m > https://docs.getdbt.com/docs/configure-your-profile
2023-01-06 15:38:04 [45mdbt[0m > 
2023-01-06 15:38:04 [45mdbt[0m > 15:38:04  Encountered an error:
2023-01-06 15:38:04 [45mdbt[0m > Runtime Error
2023-01-06 15:38:04 [45mdbt[0m >   Could not run dbt
2023-01-06 15:38:05 [32mINFO[m i.a.w.n.DefaultNormalizationRunner(close):186 - Terminating normalization process...
2023-01-06 15:38:05 [32mINFO[m i.a.w.n.DefaultNormalizationRunner(close):199 - Normalization process successfully terminated.
2023-01-06 15:38:05 [32mINFO[m i.a.w.t.TemporalAttemptExecution(lambda$getWorkerThread$5):198 - Completing future exceptionally...
io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:72) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:26) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.airbyte.workers.exception.WorkerException: DBT Transformation Failed.
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:68) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	... 3 more
	Suppressed: io.airbyte.workers.exception.WorkerException: Dbt transformation process wasn't successful
		at io.airbyte.workers.general.DbtTransformationRunner.close(DbtTransformationRunner.java:137) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
		at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:57) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
		at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:26) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
		at java.lang.Thread.run(Thread.java:1589) ~[?:?]
2023-01-06 15:38:05 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-01-06 15:38:05 [32mINFO[m i.a.c.t.TemporalUtils(withBackgroundHeartbeat):283 - Stopping temporal heartbeating...
2023-01-06 15:38:05 [33mWARN[m i.t.i.a.ActivityTaskExecutors$BaseActivityTaskExecutor(execute):114 - Activity failure. ActivityId=1ed520b8-d814-3a65-9ac7-e8c1e27936d4, activityType=Run, attempt=1
java.lang.RuntimeException: io.temporal.serviceclient.CheckedExceptionWrapper: java.util.concurrent.ExecutionException: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:281) ~[io.airbyte-airbyte-commons-temporal-0.40.26.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.run(DbtTransformationActivityImpl.java:101) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor$POJOActivityInboundCallsInterceptor.executeActivity(RootActivityInboundCallsInterceptor.java:64) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor.execute(RootActivityInboundCallsInterceptor.java:43) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskExecutors$BaseActivityTaskExecutor.execute(ActivityTaskExecutors.java:95) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskHandlerImpl.handle(ActivityTaskHandlerImpl.java:92) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handleActivity(ActivityWorker.java:241) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:206) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:179) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:93) ~[temporal-sdk-1.17.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.temporal.serviceclient.CheckedExceptionWrapper: java.util.concurrent.ExecutionException: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.temporal.serviceclient.CheckedExceptionWrapper.wrap(CheckedExceptionWrapper.java:57) ~[temporal-serviceclient-1.17.0.jar:?]
	at io.temporal.internal.sync.WorkflowInternal.wrap(WorkflowInternal.java:461) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.activity.Activity.wrap(Activity.java:52) ~[temporal-sdk-1.17.0.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:167) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$3(DbtTransformationActivityImpl.java:132) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:276) ~[io.airbyte-airbyte-commons-temporal-0.40.26.jar:?]
	... 14 more
Caused by: java.util.concurrent.ExecutionException: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:161) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$3(DbtTransformationActivityImpl.java:132) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:276) ~[io.airbyte-airbyte-commons-temporal-0.40.26.jar:?]
	... 14 more
Caused by: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:72) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:26) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	... 1 more
Caused by: io.airbyte.workers.exception.WorkerException: DBT Transformation Failed.
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:68) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:26) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
	... 1 more
	Suppressed: io.airbyte.workers.exception.WorkerException: Dbt transformation process wasn't successful
		at io.airbyte.workers.general.DbtTransformationRunner.close(DbtTransformationRunner.java:137) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
		at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:57) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
		at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:26) ~[io.airbyte-airbyte-commons-worker-0.40.26.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.40.26.jar:?]
		at java.lang.Thread.run(Thread.java:1589) ~[?:?]
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value HTTPS
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value GCS
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value S3
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value AzBlob, $.storage_account: is missing but it is required
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value SSH, $.password: object found, string expected
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value SCP, $.password: object found, string expected
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.password: object found, string expected
2023-01-06 15:38:08 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value local
2023-01-06 15:38:09 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/13/1/logs.log
2023-01-06 15:38:09 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.40.26
2023-01-06 15:38:09 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):172 - Attempt 0 to save workflow id for cancellation
2023-01-06 15:38:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:38:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- START CHECK -----
2023-01-06 15:38:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:38:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - Checking if airbyte/source-file:0.2.31 exists...
2023-01-06 15:38:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - airbyte/source-file:0.2.31 was found locally.
2023-01-06 15:38:09 [32mINFO[m i.a.w.p.DockerProcessFactory(create):120 - Creating docker container = source-file-check-13-1-znuen with resources io.airbyte.config.ResourceRequirements@1034c6e7[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
2023-01-06 15:38:09 [32mINFO[m i.a.w.p.DockerProcessFactory(create):164 - Preparing command: docker run --rm --init -i -w /data/13/1 --log-driver none --name source-file-check-13-1-znuen --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=true -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_ROLE= -e WORKER_JOB_ATTEMPT=1 -e WORKER_CONNECTOR_IMAGE=airbyte/source-file:0.2.31 -e AUTO_DETECT_SCHEMA=false -e AIRBYTE_VERSION=0.40.26 -e WORKER_JOB_ID=13 airbyte/source-file:0.2.31 check --config source_config.json
2023-01-06 15:38:09 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):97 - Reading messages from protocol version 0.2.0
2023-01-06 15:38:15 [33mWARN[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):117 - ignoring unsupported keyword arguments: ['connect_kwargs']
2023-01-06 15:38:18 [33mWARN[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):117 - ignoring unsupported keyword arguments: ['connect_kwargs']
2023-01-06 15:38:20 [32mINFO[m i.a.w.i.DefaultAirbyteStreamFactory(internalLog):120 - Check succeeded
2023-01-06 15:38:21 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-01-06 15:38:21 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:38:21 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - ----- END CHECK -----
2023-01-06 15:38:21 [32mINFO[m i.a.c.i.LineGobbler(voidCall):114 - 
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value HTTPS
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value GCS
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value S3
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value AzBlob, $.storage_account: is missing but it is required
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value SSH, $.password: object found, string expected
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value SCP, $.password: object found, string expected
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.password: object found, string expected
2023-01-06 15:38:23 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):130 - JSON schema validation failed. 
errors: $.storage: must be a constant value local`

## Steps to Reproduce
1. Setup SFTP and put a CSV file into the SFTP directory
2. Setup SQL Server as destination
3. Run sync

**Comment by BenjaminGK** [comment]
This is happening for MySQL destinations as well

**Comment by cgardens** [comment]
Closed as won't do because we have [deprecated this feature](https://github.com/airbytehq/airbyte/discussions/34860).
